### PR TITLE
stellar transaction details page tweaks

### PR DESCRIPTION
**Pull request by zanderz**
move cancel button to bottom + change text
don't show blue background in details page

### CODE REVIEW

File: shared/wallets/transaction-details/index.js [hunk 1]
```diff
@@ -359,6 +350,18 @@ const TransactionDetails = (props: NotLoadingProps) => {
             </Kb.Text>
           )}
         </Kb.Box2>
+        <Kb.Box2 direction="vertical" gap="xxtiny" fullWidth={true} style={styles.buttonBox}>
+          {props.onCancelPayment && (
```
Comment (buoyad) [inline shared/wallets/transaction-details/index.js]: Could you move this up a line to cover the `Box2` as well?

File: shared/wallets/transaction/index.js [hunk 2]
```diff
@@ -105,6 +106,7 @@ const Detail = (props: DetailProps) => {
   const textTypeSemibold = props.large ? 'BodySemibold' : 'BodySmallSemibold'
   const textTypeSemiboldItalic = props.large ? 'BodySemiboldItalic' : 'BodySmallSemiboldItalic'
   const textTypeExtrabold = props.large ? 'BodyExtrabold' : 'BodySmallExtrabold'
+  const textSentenceEnd = props.detailView && props.pending ? '\u2026' : '.'
```
Comment (buoyad) [inline shared/wallets/transaction/index.js]: :sheep: a comment saying `\u2026` is an ellipsis would be nice

File: shared/wallets/transaction-details/index.js (hunk 1 above)
Comment (zanderz) [reply shared/wallets/transaction-details/index.js]: ok

File: shared/wallets/transaction/index.js (hunk 2 above)
Comment (zanderz) [reply shared/wallets/transaction/index.js]: does that mean you are asking sheepishly?

Comment (buoyad) [reply shared/wallets/transaction/index.js]: Lol, it's a suggestion but not a deal breaker

Comment (zanderz) [reply shared/wallets/transaction/index.js]: just wondering what sheep means

**Pull request merged**